### PR TITLE
fix: add run command in project root path, make hindent can load .hin…

### PR DESCRIPTION
**Pull request by rebornwwp**

### CODE REVIEW

**Comment by rebornwwp** [comment]
Hello, I found your extensions can't find hindent custom config file.   So I put all command runing in the project root path so that hindent can found this file. need your review thanks

File: src/extension.ts [hunk 1]
```diff
@@ -36,17 +36,18 @@ export function activate(context: vscode.ExtensionContext) {
 
 				let result = document.getText();
 				const commentSpaces = checkCommentSpaces(result);
+				const cwd = vscode.workspace.rootPath;
```
Comment (miloszwasacz) [inline src/extension.ts]: `rootPath` is deprecated. Maybe use `const cwd = vscode.workspace.workspaceFolders?.at(0)?.uri?.fsPath;` instead?

**Review by miloszwasacz — CHANGES_REQUESTED** [review]
The code looks good apart from the usage of `vscode.workspace.rootPath`, which is deprecated. I suggested a non-deprecated alternative. Fix that and we can merge you PR.

File: src/extension.ts (hunk 1 above)
Comment (rebornwwp) [reply src/extension.ts]: ok, I will fix this recently.

**Comment by rebornwwp** [comment]
I have fix this problem. But the `at` method is always  with the error `'at' does not exits on type 'readonly string []'`. Is there a way to make the code more simple?

**Review by miloszwasacz — APPROVED** [review]

**Pull request merged**